docs/src/main/asciidoc/spring-di.adoc

```diff
@@ -107,11 +186,12 @@ public class AppConfiguration {
 }
 ----
 
-Now we define another bean that will implement `StringFunction` using Spring's stereotype annotation style.
-This bean will effectively be a no-op bean that simply returns the input as is.
-Create a `src/main/java/org/acme/spring/di/NoOpSingleStringFunction.java` file and set the following content:
-
-[source,java]
+. Create a `NoOpSingleStringFunction` bean inside the the `src/main/java/org/acme/spring/di/NoOpSingleStringFunction.java` file.
+The `NoOpSingleStringFunction` implements the `StringFunction` interface using the `@Component` stereotype annotation provided by Spring.
```

Comment:
This explanation is not as clear as what was before. 

The details of which annotations aren't as relevant as that this is a noop. 

"Returning without processing" is complicated way of saying that imo.

Reply:
OK, thank you, I will take your advice on this, and change the wording to be closer to the original.